tools/rapids-configure-conda-channels

```diff
@@ -1,11 +1,29 @@
 #!/bin/bash
 # A utility script that configures conda channels
 
+conda_channel_in_config() {
+  channel_id=${1:?err}
+  conda config --json --get channels \
+  | jq -r --arg c "${channel_id}" '."get"."channels" | any(. == $c )'
```

Comment:
`jq` has a `--exit-status`/`-e` flag for converting booleans to exit codes.

if you use that flag here, then you can update the calling lines to just look like this:

```sh
if conda_channel_in_config "${channel_id}"; then
# ...
```

also, `jq` doesn't require double quotations when you're accessing keys.

e.g. `'."get"."channels"'` can be just `'.get.channels'`, which helps with readability.

finally, you can avoid having to pass `--arg c` by exporting `channel_id` (e.g. `export channel_id=${1:?err}`) and using `env.channel_id` in `jq` like this:

```sh
jq '.get.channels | any(. == env.channel_id)'
```

Reply:
Thanks, did not know about `env.{whatever}`!

I just pushed some changes implementing most of this. The one difference is I'm proposing this:

```shell
channel_id="${channel_id}" jq ...
```

instead of

```shell
export channel_id="${channeld_id}"
jq ...
```

So that function doesn't have side-effects observable outside of it (e.g. setting `channel_id` in global scope).